apps/video/index.html

```diff
@@ -51,12 +51,12 @@
     <div id="app-container">
       <div id="two-column-spearator"></div>
       <section role="region" id="thumbnail-views">
-        <gaia-header ignore-dir id="picker-header" action="back" class="thumbnails-list">
+        <gaia-header ignore-dir id="picker-header" action="back" class="thumbnails-list" no-font-fit>
```

Comment:
I didn't the add 'title-start' and 'title-end' attributes because I found the header title wasn't being centered properly. I investigated and found the gaia-header logic wasn't using the correct screen width when these attributes were supplied. When they were not supplied, the width was determined properly.